website/views/shared/header/header.jade

```diff
@@ -31,9 +31,10 @@
           | {{Math.floor(user.stats.mp)}} / {{user._statsComputed.maxMP}}
     // party
     .party(ng-controller='PartyCtrl')
-      button.party-invite.btn.btn-primary(ng-click="inviteOrStartParty(group)",
-        ng-if="(!party.members || party.memberCount === 1) && user.preferences.displayInviteToPartyWhenPartyIs1",
-        popover="{{!party.members ? env.t('startAParty') : env.t('addToParty')}}", popover-placement="left", popover-trigger="mouseenter")
-        span=env.t("battleWithFriends")
       .herobox-wrap(ng-repeat='profile in partyMinusSelf')
         +herobox()
+      .party-invite-wrap
+        button.party-invite.btn.btn-primary(ng-click="inviteOrStartParty(group)",
+          ng-if="(!party.members || party.memberCount < 4 ) && user.preferences.displayInviteToPartyWhenPartyIs1",
```

Comment:
I've looked into what's required for a migration to change the preference name from `displayInviteToPartyWhenPartyIs1` and it should be simple. I can run one migration script shortly before this change goes live to create the new preference option in all user accounts, and then after go-live, run another to remove the old option. I've updated the first comment in the PR to add a reminder that I need to do this.